vagrant/setup.sh

```diff
@@ -30,6 +30,24 @@ Argument:
 EOF
 }
 
+function enable_iommu {
+    echo "Checking Intel iommu"
+    if ! find /sys/kernel/iommu_groups/*/devices/* > /dev/null 2>&1; then
+        echo "Enabling Intel_iommu"
+        sudo sed -i "s|^GRUB_CMDLINE_LINUX\(.*\)\"|GRUB_CMDLINE_LINUX\1 intel_iommu=on\"|g" /etc/default/grub
+        grub_cfg=$(sudo readlink -e /etc/grub2.cfg)
+        if sudo [ -f ${grub_cfg} ]; then
```

Comment:
Could you surround the `${grub_cfg}` var with strings to pass shellcheck issues

```
In setup.sh line 39:
        if sudo [ -f ${grub_cfg} ]; then
                     ^---------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
        if sudo [ -f "${grub_cfg}" ]; then
```